internal_ws/yktrace/src/swt.rs

```diff
@@ -147,9 +159,9 @@ __yk_swt_rec_loc_active:
 .att_syntax prefix
 "#
 );
+*/
 
 /// This is outlined to make the common case of tracing being disabled faster.
```

Comment:
Not your change, but I think this is supposed to say "designed", not "outlined".

Reply:
Outlining is the opposite operation to inlining. With inlining you take the code of one function and put it at the call site. With outlining you take a part of a function and put it in a new function you call.

Reply:
I've never heard that term before, but makes sense!